django/utils/six.py

```diff
@@ -737,6 +794,25 @@ def wrapper(cls):
         return metaclass(cls.__name__, cls.__bases__, orig_vars)
     return wrapper
 
+
+def python_2_unicode_compatible(klass):
```

Comment:
Should we use this in `django.utils.encoding`? Looks like the implementation is same as https://github.com/django/django/blob/master/django/utils/encoding.py#L27-42